src/libs/actions/User.js

```diff
@@ -78,17 +76,23 @@ function updatePassword(oldPassword, password) {
  * @param {String} message optional reason for closing account
  */
 function closeAccount(message) {
-    DeprecatedAPI.User_Delete({message}).then((response) => {
-        console.debug('User_Delete: ', JSON.stringify(response));
-
-        if (response.jsonCode === 200) {
-            Growl.show(Localize.translateLocal('closeAccountPage.closeAccountSuccess'), CONST.GROWL.SUCCESS);
-            redirectToSignIn();
-            return;
-        }
-
-        // Inform user that they are currently unable to close their account
-        CloseAccountActions.showCloseAccountModal();
+    // Note: successData does not need to set isLoading to false because if the CloseAccount
+    // command succeeds, a Pusher response will clear all Onyx data.
```

Comment:
Honestly seems unecessary as this will eventually apply to so many other API calls. But it's not a blocker

Reply:
Wait really? I thought it was useful b/c I usually see `successData` also clearing any loading state - I didn't know there would be many API calls clearing all Onyx state?